node/pegnet/txbatchholding.go

```diff
@@ -0,0 +1,85 @@
+package pegnet
+
+import (
+	"database/sql"
+	"github.com/pegnet/pegnetd/fat/fat2"
+
+	"github.com/Factom-Asset-Tokens/factom"
+)
+
+// createTableTransactionBatchHolding is a SQL string that creates the
+// "pn_transaction_batch_holding" table.
+//
+// The "pn_transaction_batch_holding" table has a foreign key reference to the
```

Comment:
I think this comment is not needed?

Reply:
Yeah sorry its a remnant of my copypasting that i didn't remove :grin: